std/algorithm/comparison.d

```diff
@@ -211,6 +211,13 @@ Throws: If none of the choice matches, a $(D SwitchError) will be thrown.  $(D
 SwitchError) will also be thrown if not all the choices are void and a void
 choice was executed without throwing anything.
 
+Params:
+    choices = The choices to select from.
+    switchObject = The $(D Object) compared to the choices.
```

Comment:
If for every 10 posts in that thread a docu PR would have been created phobos docu would be the best docu in the world. I already hate myself writing this two comments